packages/jest-core/src/cli/index.ts

```diff
@@ -134,6 +134,7 @@ const buildContextsAndHasteMaps = async (
       createDirectory(config.cacheDirectory);
       const hasteMapInstance = Runtime.createHasteMap(config, {
         console: new CustomConsole(outputStream, outputStream),
+        enableSymlinks: config.haste.enableSymlinks,
```

Comment:
We don't need to pass this, do we? We pass the full `config` as the first argument

Reply:
Good point, done